src/aurora/mod.rs

```diff
@@ -440,9 +440,6 @@ where
         }
 
         // ======================== Univariate sumcheck test ========================
-        let zero_padded_instance =
-            [instance.clone(), vec![F::ZERO; num_witness_variables]].concat();
```

Comment:
notice that the clone was previously happening here.

though now that I look at this, we could have resized to the full new size in one go, and avoid the clone altogether?

Reply:
Hmm what do you exactly mean by "resize to the full new size in one go, and avoid the clone altogether"? What kind of code do you have in mind? `resize` doesn't clone, so whether to clone or not seems independent of resizing vs. concatenating with [0, ..., 0], right?

Reply:
Cf. long message below ^^